src/internal.rs

```diff
@@ -351,6 +351,7 @@ impl Rav1dFrameContext_bd_fn {
     }
 }
 
+#[derive(Default, Clone, Copy)]
```

Comment:
```suggestion
#[derive(Default)]
```

Both the `Box<[_]>` and `Vec<_>` can be written without `Clone` or `Copy`, so I wouldn't add them for that.